### PR TITLE
GHC-136: Add lab ranges and accept decimals

### PR DESCRIPTION
**Pull request by basrem**

### CODE REVIEW

File: configuration/concepts/lab_tests.csv [hunk 1]
```diff
@@ -34,35 +34,35 @@ a360af3a-3797-431e-a1a4-b8dbb839f20e,,,Monocyte,Monocitos,Monocitos,Monocyte,,La
...
 560fa493-f0d5-4a5f-8623-870f28cd0e61,,,Eosinophil,Eosinófilos,Eosinófilos,Eosinophil,,LabTest,Numeric,,,,,,,,,,
 881fdefd-e5ac-4131-a01f-a3208400f73f,,,Creatinine,Creatinina,Creatinina,Creatinine,,LabTest,Numeric,,,,Yes,mg/dl,0.5,1.1,,,
-931c283b-efc4-479e-97c6-a3e7079ab11b,,,BUN,BUN,BUN,BUN,,LabTest,Numeric,,,,,mg/dl,7,20,,,
-3021d3e6-14ce-4357-afa0-9fbab4953326,,,Urea,Urea,Urea,Urea,,LabTest,Numeric,,,,,mg/dl,10,50,,,
-2372438b-3441-4fe2-bb4c-37f3b04b525e,,,Sodium,Sodio,Sodio,Sodium,,LabTest,Numeric,,,,,mEq/g,138,148,,,
-41857888-2814-43c0-b3b4-3bb36975342c,,,Potassium,Potasio,Potasio,Potassium,,LabTest,Numeric,,,,Yes,mEq/g,3.6,4.2,,,
-5c86bce9-9bd0-413b-a0de-bdaa1f05a0df,,,Calcium Test,Calcio,Calcio,Calcium,,LabTest,Numeric,,,,,,,,,,
+931c283b-efc4-479e-97c6-a3e7079ab11b,,,BUN,BUN,BUN,BUN,,LabTest,Numeric,,,,Yes,mg/dl,7,20,,,
+3021d3e6-14ce-4357-afa0-9fbab4953326,,,Urea,Urea,Urea,Urea,,LabTest,Numeric,,,,Yes,mg/dl,10,50,,,
+2372438b-3441-4fe2-bb4c-37f3b04b525e,,,Sodium,Sodio,Sodio,Sodium,,LabTest,Numeric,,,,,mEq/L,138,148,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: Should this also accept decimal places?

Comment (basrem) [reply configuration/concepts/lab_tests.csv]: Yes it should, fixed it below

**Review by reagan-meant — CHANGES_REQUESTED** [review]
Use the old uuid of the form

File: configuration/concepts/lab_tests.csv [hunk 2]
```diff
@@ -99,7 +99,7 @@ f87891da-16d1-4e15-afdf-aa114ab616bf,,,Other culture,Otra cultura,Otra cultura,O
 d3966267-5882-437b-ae14-18dee9657d14,,,RBCs,Hematíes,Hematíes,RBCs,,LabTest,Coded,,Positive; Negative,,,,,,,,
 1b985633-13de-4afa-828e-ef38721c660e,,,Ferning,Helecho,Helecho,Ferning,,LabTest,Coded,,Positive; Negative,,,,,,,,
 dbb2a948-ab68-4474-a24b-5f0ecf697f88,,,Whiff Test,Prueba de olor,Prueba de olor,Whiff Test,,LabTest,Coded,,Positive; Negative,,,,,,,,
-acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,Aspect;Color;PH; Specific gravity; Glucose; Ketones; Urobilinogen; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,,,,,,,
+acdc2f09-2844-43e0-9067-8e71da8319ae,,,Urinalysis,Análisis de orina,Análisis de orina,Urinalysis,,LabSet,N/A,,,Aspect;Color;PH; Specific gravity; Glucose; Ketones; Urobilinogen; Nitrites; Leukocyte esterase; Bilirubin; Blood in urine; Protein,Yes,,,,,,
```
Comment (reagan-meant) [inline configuration/concepts/lab_tests.csv]: What is the `Yes` for here?

**Review by reagan-meant — CHANGES_REQUESTED** [review]

**Pull request merged**